.github/workflows/create-docs-issues.js

```diff
@@ -13,7 +13,7 @@ module.exports = async ({ github, context }) => {
     github.rest.issues.create({
       owner: context.repo.owner,
       repo: ISSUE_REPO_NAME,
-      title: `${type.toUpperCase()}: ${context.payload.pull_request.title}`,
+      title: `${context.payload.pull_request.title}`,
```

Comment:
We don't want any sort of prefix in the title?

Reply:
Nope. Margaret requested just the 1 ticket w/ no prefix and adding the `Community` label which will be added since it is the `type`.